multinet/types.py

```diff
@@ -1,9 +1,63 @@
 """Custom types for Multinet codebase."""
-from typing import Dict, Set
+from pydantic import BaseModel, Field
+from typing import List, Dict, Set, Optional, Any
 from typing_extensions import Literal, TypedDict
 
 EdgeDirection = Literal["all", "incoming", "outgoing"]
-TableType = Literal["all", "node", "edge"]
+
+TableType = Literal["node", "edge"]
+UnionTableType = Literal["all", TableType]
```

Comment:
I checked the codebase and `TableType` isn't used anywhere. So I think it makes sense to just add `"all"` to `TableType` and not bother with this new `UnionTableType` (which sounds like it sets out the types for a thing called a "union table").

Reply:
Whoops, this is leftover from my initial approach. I actually removed [this](https://github.com/multinet-app/multinet-rfcs/blob/91534820bcdcd620eefc15dff8a54f0fdeb0900c/accepted/0009-table-metadata/README.md#L31) line from the implementation, as it seemed redundant and just got in the way. I'll update the RFC and revert this change.